RadixWallet/Features/ProfileBackupsFeature/RestoreProfileFromBackup/Children/RecoverWalletWithoutProfile/Children/Start/RecoverWalletWithoutProfileStart+View.swift

```diff
@@ -22,36 +22,43 @@ extension RecoverWalletWithoutProfileStart {
 			WithViewStore(store, observe: \.viewState, send: { .view($0) }) { _ in
 				ScrollView {
 					VStack(alignment: .center, spacing: .large3) {
+						// FIXME: Strings
 						Text("Recover Control Without Backup")
 							.textStyle(.sheetTitle)
 
+						// FIXME: Strings
 						Text("If you have no wallet backup in the cloud or as an exported backup file, you still have other restore options.")
 							.multilineTextAlignment(.leading)
-							.textStyle(.body1Regular)
 
 						Divider()
 
+						// FIXME: Strings
 						Text("**I have my main “Babylon” 24-word seed phrase.**")
 
+						// FIXME: Strings
 						Button("Recover with Main Seed Phrase") {
 							store.send(.view(.recoverWithBDFSTapped))
 						}
 						.buttonStyle(.secondaryRectangular)
 
 						Divider()
 
+						// FIXME: Strings
 						Text("**I only want to restore Ledger hardware wallet Accounts**")
 
+						// FIXME: Strings
 						Text("OR")
 
+						// FIXME: Strings
 						Text("**I only have Accounts created on the Radix Olympia Network**")
 
-						Button("Ledger-only or Olmypia-only Restore") {
+						// FIXME: Strings
+						Button("Ledger-only or Olympia-only Restore") {
 							store.send(.view(.ledgerOnlyOrOlympiaOnlyTapped))
 						}
 						.buttonStyle(.secondaryRectangular)
```

Comment:
When and where is this actually shown? It looks quite unusual.

Reply:
![IMG_0661](https://github.com/radixdlt/babylon-wallet-ios/assets/864410/697f05ad-ae53-4cef-8edb-214b43b349ab)

In Onboarding, from selecting "Other Restore Option" in SelectBackup.